### ISSUE TITLE
Completed Download Handling Not Linking to a Wanted Movie

### ISSUE THREAD

**Issue by nicholasfoote** [Radarr/Radarr#201]
Once a download in NZBget is done, it's not importing automatically because it isn't linking to a wanted movie. I am able to click on the blank movie field and choose the correct movie and it imports successfully, but it should be able to do this on it's own.

![image](https://cloud.githubusercontent.com/assets/10204046/21903575/45b953dc-d8b5-11e6-9fe8-aa4bbf4a5861.png)

**Comment by onedr0p** [comment]
I wonder why it wasn't picked up for automation, can you post logs?

**Comment by nicholasfoote** [comment]
Here are logs for a fresh download I grabbed. Is that site supposed to be resolvable in a browser? I tried multiple locations/networks and only get 404s.

Component	Message	Time
RssSyncService	RSS Sync Completed. Reports found: 200, Reports grabbed: 0	9:37am

DownloadDecisionMaker	Processing 200 releases	9:37am

RssSyncService	Starting RSS Sync	9:37am

**SceneMappingService	Failed to Update Scene Mappings:: HTTP request failed: [404:NotFound] [GET] at [http://radarr.aeonlucid.com/v1/scenemapping]	9:36am**

**HttpClient	HTTP Error - Res: [GET] http://radarr.aeonlucid.com/v1/scenemapping: 404.NotFound	9:36am**

SceneMappingService	Updating Scene mappings	9:36am

Database	Main database compressed	9:34am
Database	Vacuuming Main database	9:34am
Database	Log database compressed	9:34am
Database	Vacuuming Log database	9:34am
HousekeepingService	Running housecleaning tasks	9:34am

DownloadService	Report sent to NZBget. Alone.in.Berlin.2016.1080p.WEB-DL.DD5.1.H264-FGT	9:34am

Nzbget	Adding report [Alone.in.Berlin.2016.1080p.WEB-DL.DD5.1.H264-FGT] to the queue.	9:34am

DownloadDecisionMaker	Processing 7 releases	9:34am

NzbSearchService	Searching 2 indexers for [Alone in Berlin]	9:34am
